jsdynamic-typify/src/test/scala/CanParseProp.scala

```diff
@@ -0,0 +1,50 @@
+package typify
+
+import scala.scalajs.js
+import scala.scalajs.js.Dynamic.literal
+import scala.scalajs.js.annotation.JSExport
+import scala.scalajs.js.typify.parsedinstances._
+import org.scalacheck.Properties
+
+object MakeJsDynamic extends MakeParsed[js.Dynamic] {
+
+  import implicits._
+
+  def make[A](k: String, v: A)(implicit mp: MustParse[A]): js.Dynamic =
+    mp match {
+      case MPS => literal(k -> v)
+      case MPOS => MPOS(v).map(x => literal(k -> x)).getOrElse(().asInstanceOf[js.Dynamic])
+      case MPI => literal(k -> v)
+      case MPOI => MPOI(v).map(x => literal(k -> x)).getOrElse(().asInstanceOf[js.Dynamic])
+      case MPL => literal(k -> v)
+      case MPOL => MPOL(v).map(x => literal(k -> x)).getOrElse(().asInstanceOf[js.Dynamic])
+      case MPP => literal(k -> v)
+      case MPOP => MPOP(v).map(x => literal(k -> x)).getOrElse(().asInstanceOf[js.Dynamic])
+    }
+  def to[A](v: A)(implicit mp: MustParse[A]): js.Dynamic =
+    mp match {
+      case MPS => v.asInstanceOf[js.Dynamic]
+      case MPOS => MPOS(v).map(_.asInstanceOf[js.Dynamic])
+                    .getOrElse(().asInstanceOf[js.Dynamic])
+      case MPI => v.asInstanceOf[js.Dynamic]
+      case MPOI => MPOI(v).map(_.asInstanceOf[js.Dynamic])
+                    .getOrElse(null.asInstanceOf[js.Dynamic])
+      case MPL => v.asInstanceOf[js.Dynamic]
+      case MPOL => MPOL(v).map(_.asInstanceOf[js.Dynamic])
+                    .getOrElse(().asInstanceOf[js.Dynamic])
+      case MPP => v.asInstanceOf[js.Dynamic]
+      case MPOP => MPOP(v).map(_.asInstanceOf[js.Dynamic])
+                    .getOrElse(null.asInstanceOf[js.Dynamic])
```

Comment:
Probably should `Gen.choose(null, ()).sample.getOrElse(())` for all getOrElse branches.